fe/i18n/messages/en.json

```diff
@@ -30,6 +30,10 @@
     },
     "EventInfo": {
       "level": "Level:"
+    },
+    "Stepper": {
+      "textOne": "PAYMENT INFO",
```

Comment:
Can you change the text to camelcase instead of capslock?

Reply:
But to be on the screen with Camelcase?

Reply:
No, but i think it makes more sense for the translations to be camelcase and for the component to be uppercase for any text that comes in.